.github/scripts/generate-os-packages.sh

```diff
@@ -133,11 +148,13 @@ generate_sdk() {
 }
 
 if [[ "$OSTYPE" == "linux-gnu"* ]]; then
-  generate_deb
-  generate_rpm
+  if [ "$ARCHITECTURE" == "x86_64" ]; then
+    generate_deb
+    generate_rpm
+  fi
```

Comment:
We don't need to generate deb and rpm files when running inside the aarch64 QEMU vm.